metrics-operator/controllers/common/providers/common.go

```diff
@@ -4,6 +4,7 @@ const DynatraceProviderType = "dynatrace"
 const DynatraceDQLProviderType = "dql"
 const PrometheusProviderType = "prometheus"
 const ThanosProviderType = "thanos"
+const CortexProviderType = "cortex"
 const DataDogProviderType = "datadog"
 
 var SupportedProviders = []string{
```

Comment:
I believe we also must add this new provider (and `thanos` as well) to this array, as this is used to create the channels for the analysis value requests: https://github.com/keptn/lifecycle-toolkit/blob/7cc1622edc1c7e6ffa138389df2b7e15f1c2b8db/metrics-operator/controllers/analysis/provider_selector.go#L39

Reply:
nice catch!